internal/pkg/aws/ecs/ecs.go

```diff
@@ -168,6 +181,60 @@ func (e *ECS) ServiceTasks(clusterName, serviceName string) ([]*Task, error) {
 	return tasks, nil
 }
 
+// DefaultClusters returns the default clusters in the account and region
```

Comment:
typo and comments should end with a period: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
```suggestion
// DefaultCluster returns the default clusters in the account and region.
```